test/sql-server-cdc-old-syntax/types-unsupported.td

```diff
@@ -0,0 +1,11 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# This file is not necessary for SQL Server but included for completeness.
```

Comment:
I think removing it is better, runs faster

Reply:
we should remove the empty mysql files with $nop in them then.